docs/EN_US/ContainerizedHPCC/ContainerizedMods/ConfigureValues.xml

```diff
@@ -1974,10 +2090,10 @@ global:
 
         <para>HPCC Systems components are added or modified by passing in
         override values. The Helm chart values are overridden, either by
-        passing in override values file(s) using -f, (for override file) or
-        via --set where you can override a single value. Those passed in
-        values are always merged in the same order they are given on the helm
-        command line.</para>
+        passing in override value file(s) using -f, (for override file) or via
+        --set where you can override a single value. Those passed in values
+        are always merged in the same order they are given on the helm command
+        line.</para>
```

Comment:
P/V 
I suggest:
Override the Helm chart values either by passing in one or more override value file(s) using -f <filename> or by using the --set option to override a single value.